README.md

```diff
@@ -1,4 +1,52 @@
-## Setup the Database
+# Running the App
+The repository contains Docker configuration which can be used out of the box, however, the app can be run without using
+the docker container. Both way are discussed below:
+
+## 1. Run inside Docker Container
+It is the recommended way, the Docker Image of the app is pre-configured to run the `migrations` and `collectstatic`.
+However, the `load` command to populate the database and the crons need to be run manually.
+
+The image can be built locally or can be pulled from Docker Hub:
+
+```
+docker pull arjunsalyan/macports-webapp
+```
+or build the image
+```
+docker build -t macports-webapp .
+```
+
+After the image has been build or pulled, run it using an an env file that contains environment variables. Create a file
+`env` in the root of the project and supply the following information.
+
+**env**: *(Contains environment variables)*
+```
+DB_NAME=
+DB_USER=
+DB_PASSWORD=
+DB_HOST=
+SECRET_KEY=
+```
+
+Now run the image:
+```
+docker run -d -p 80:80 --env-file=env macports-webapp
```

Comment:
There can be a ton of options we might need to add here later but one option which comes to my mind is `--log-opt max-size=1g` to remove the older logs and stop filling up the disk space on the server.
[Link](https://docs.docker.com/config/containers/logging/json-file/).

Reply:
Yes, thank you.